graph-db/serverless/src/utils/cmr/indexPlatform.js

```diff
@@ -0,0 +1,99 @@
+import gremlin from 'gremlin'
+
+const gremlinStatistics = gremlin.process.statics
+
+/**
+ * create the acquiredBy edge between platform/instrument and collection
+ * @param {JSON} piVertex the platformInstrument vertex
+ * @param {Connection} gremlinConnection a connection to the gremlin server
+ * @param {Graph Node} dataset the parent collection vertex in the gremlin server
+ * @returns null
+ */
+const createAcquiredByEdge = async (piVertex, gremlinConnection, dataset) => {
```

Comment:
Normally each of these functions would be their own file to make testing easier. I think it's worth sticking to that pattern

Reply:
I think it is more readable to keep this private function in this file.

Reply:
I disagree because it's an inconsistency in how we organize our code

Reply:
I agree with @mschmele

Reply:
Reluctantly refactored code. :)